pallets/gear/src/tests.rs

```diff
@@ -408,10 +408,10 @@ fn block_gas_limit_works() {
 
```

Comment:
I would recommend to calculate it from unused user for tests, to prevent future errors and test issues